en/docs/quick-start-guides/deploy-your-first-automation.md

```diff
@@ -0,0 +1,127 @@
+# Deploy Your First Automation
+
+## Overview
+
+In this guide, you will:
+
+- Create a Devant project and add a simple automation which will print `"Hello World"` every minute .
+- Use Ballerina Integrator (BI) to develop the automation.
+- Push the automation to Devant from the BI which will automatically build the automation.
+- Schedule the automation to run every minute.
+
+<!-- Todo add a youtube video tutorial for this quick start -->
+
+## Prerequisites
+
+1. GitHub account: [Create a github account](https://github.com/signup) if you don't have one already.
+<!-- [//]: # (Todo check the url is correct before going live) -->
+2. If you're signing in to the Devant for the first time, create an organization:
+    1. Go to [https://devant.dev/](https://devant.dev/) and sign in using your Google, GitHub, or Microsoft account.
+    2. Enter a unique organization name. For example, `Stark Industries`.
+    3. Read and accept the privacy policy and terms of use and Click **Create**.
+
+1. VSCode: [Install VSCode](https://code.visualstudio.com/download) if you don't have it installed already.
+2. Create a profile in VSCode:
```

Comment:
Say
`[Only needed if you have a pre-existing Ballerina installation]`

Reply:
Profile requirements will go away when we go GA in BI. lets remove it.

Reply:
Removed